config/config/wc-config.yaml

```
@@ -156,80 +302,152 @@ ck8sdash:
         memory: 128Mi
         cpu: 100m
 
+## Set external traffic policy to: "Local" to preserve source IP on
+## providers supporting it
+## Ref: https://kubernetes.io/docs/tutorials/services/source-ip/#source-ip-for-services-with-typeloadbalancer
 externalTrafficPolicy:
   local: false
+
+  ## Source IP range to allow.
+  ##
   whitelistRange:
     global: "0.0.0.0/0"
     ck8sdash: false
 
+## Nfs-client-provisioner configuration
+##
 nfsProvisioner:
+  ## Define resources requests and limits for single Pods.
+  ## ref: https://kubernetes.io/docs/user-guide/compute-resources/
+  ##
   resources:
     limits:
       cpu: 100m
       memory: 128Mi
     requests:
       cpu: 100m
       memory: 128Mi
+
+  ## Tolerations for pod assignment
+  ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+  ##
   tolerations: []
-  affinity: {}
+
+  ## Node labels for pod assignment
+  ## ref: https://kubernetes.io/docs/user-guide/node-selection/
+  ##
   nodeSelector: {}
 
+  ## Affinity for pod assignment
+  ## ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity
+  ##
+  affinity: {}
+
+## Nginx ingress controller configuration
+##
 ingressNginx:
   controller:
-  # Chart deploys correctly but does not work with resourceRequests
+    ## Define resources requests and limits for single Pods.
+    ## ref: https://kubernetes.io/docs/user-guide/compute-resources/
+    ##
     resources: {}
-      # limits:
-      #   cpu: 100m
-      #   memory: 64Mi
-      # requests:
-      #   cpu: 100m
-      #   memory: 64Mi
+
+    ## Tolerations for pod assignment
+    ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+    ##
     tolerations:
       - key: "nodeType"
         operator: "Exists"
         effect: "NoSchedule"
+
+    ## Affinity for pod assignment
+    ## ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity
+    ##
     affinity: {}
+
+    ## Node labels for pod assignment
+    ## ref: https://kubernetes.io/docs/user-guide/node-selection/
+    ##
     nodeSelector: {}
+
     config:
       useProxyProtocol: "set-me"
+
+    ## Use host ports 80 and 443
```

```suggestion
    ## Use host ports 80 or 443
```
You can't use both right?

---

https://github.com/kubernetes/ingress-nginx/blob/master/charts/ingress-nginx/values.yaml#L56-L63

---

I misunderstood the comment. I thought it meant that I should set `useHostPort: 80` or `useHostPort: 443`. But here I see that it should be `true` or `false`: https://github.com/elastisys/compliantkubernetes-apps/blob/15038efd79296dde6e996233e88dbfc096dd6a0a/helmfile/values/ingress-nginx.yaml.gotmpl#L35

Maybe change it to this (or something similar):
```suggestion
    ## If `true`, nginx will use host ports 80 and 443
```